sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/AggregateEstimation.scala

```diff
@@ -52,7 +52,13 @@ object AggregateEstimation {
         outputRows.min(childStats.rowCount.get)
       }
 
-      val outputAttrStats = getOutputMap(childStats.attributeStats, agg.output)
+      val inputAttrStats = childStats.attributeStats
```

Comment:
can we add a method in `EstimationUtils` in order to avoid the code redundancy?

Reply:
Done. Also added unit test case.